pkgs/collection/lib/src/iterable_extensions.dart

```diff
@@ -71,10 +70,15 @@ extension IterableExtension<T> on Iterable<T> {
   /// Creates a sorted list of the elements of the iterable.
   ///
   /// The elements are ordered by the natural ordering of the
-  /// property [keyOf] of the element.
-  List<T> sortedBy<K extends Comparable<K>>(K Function(T element) keyOf) {
+  /// [keyOf] property when [ascending] is `true` or reverse ordering
```

Comment:
The `sorted` function should get the same behavior. Really, every `sort` function should behave the same way if at all possible.
(It's probably impossible because there is *some* `sort([int Function(T, T) compare])` somewhere which prevents a named parameter.)